.github/workflows/windows.yml

```diff
@@ -0,0 +1,33 @@
+name: windows
+
+on: [push]
+
+env:
+  NUGET_URL: https://dist.nuget.org/win-x86-commandline/v5.4.0/nuget.exe
+
+jobs:
+  windows_build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Download Nuget.exe
+      run: Invoke-WebRequest -OutFile Nuget.exe $Env:NUGET_URL
+    - name: Install Dependencies
+      run: ./Nuget.exe restore App8\App8.sln
+    - name: setup-msbuild
+      uses: microsoft/setup-msbuild@v1
+    - name: Build .Wasm project
+      run: msbuild App8\App8.Wasm\App8.Wasm.csproj  /t:publish /p:PublishDir="publish_result\\"
+    - name: Copy resources & Cleanup
+      shell: pwsh
+      run : |
+        xcopy /e /k /h /i browserExtensionAssets App8\App8.Wasm\publish_result
+        cd App8\App8.Wasm\publish_result
+        (Get-Content index.html) -replace '</hrad>', '<link rel="stylesheet" type="text/css" href="./stylesheet.css" /> </head>' | Out-File index.html
+        Get-ChildItem -Filter "_*" -Recurse -Force | Remove-Item -Recurse -Force
```

Comment:
let's replace this mini script with an .fsx script so that it's cross platform :)